backend/cypress/support/dissemination-table.js

```diff
@@ -4,7 +4,9 @@
 
 const API_GOV_JWT = Cypress.env('API_GOV_JWT') || '';
 const API_GOV_KEY = Cypress.env('API_GOV_KEY') || '';
+const API_GOV_USER_ID_ADMIN = Cypress.env('API_GOV_USER_ID_ADMIN');
 const API_GOV_URL = Cypress.env('API_GOV_URL');
+const ADMIN_API_VERSION = Cypress.env('ADMIN_API_VERSION');
```

Comment:
@jadudm @danswick @eazapata23 

heads up that these are new environment vars that we'll need to get added to GHA. 

The admin user can be set to [any of these](https://github.com/GSA-TTS/FAC/blob/1af236093cab16beb783eec4021b162f04c90840/backend/support/api/admin_api_v1_0_0/create_access_tables.sql#L20-L46)

Reply:
These env vars do they needed to be added to github workflows directory? Also, do we need add this to our env vars as well?

Reply:
@eazapata23 yep!